packages/webdriverio/src/commands/element/addValue.ts

```diff
@@ -1,9 +1,22 @@
-import { transformToCharString } from '../../utils'
+import { toUnicodeCharacterArray } from '../../utils'
 
-export type AddValueOptions = {
+export type CommandOptions = {
     translateToUnicode?: boolean
 }
 
+export type Value = string | number
+
+function isValidType(value: unknown) {
+    const isNumberOrString = (_: unknown) => typeof _ === 'string' || typeof _ === 'number'
```

Comment:
let's give the parameter a proper name
```suggestion
    const isNumberOrString = (value: unknown) => typeof value === 'string' || typeof value === 'number'
```